### PR TITLE
[UX2.0] CLI feature profile and CLI config profile parcel support

### PR DESCRIPTION
**Pull request by tzarski0**
Adding support for CLI feature profile and CLI config profile parcel

### CODE REVIEW

**Comment by tzarski0** [comment]
@danischm could you please help me understand why tests are failing?

**Comment by tzarski0** [comment]
Tests fixed, ready for merge.

File: sdwan_profile_parcels.tf [hunk 1]
```diff
@@ -0,0 +1,11 @@
+resource "sdwan_cli_config_profile_parcel" "cli_config_profile_parcel" {
+  for_each = {
+    for cli in try(local.feature_profiles.cli_profiles, {}) :
+    "${cli.name}-config" => cli
+    if lookup(cli, "config", null) != null
```
Comment (danischm) [inline sdwan_profile_parcels.tf:5]: We typically use try() instead of lookup() here:

```
if try(cli.config, null) != null
```

**Review by danischm — CHANGES_REQUESTED** [review]

File: sdwan_profile_parcels.tf [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  }
+  name               = each.value.config.name
```
Comment (danischm) [inline sdwan_profile_parcels.tf:7]: Should we maybe make this name optional and use `"${each.value.name}-config"` by default?

**Review by danischm — CHANGES_REQUESTED** [review]

File: sdwan_profile_parcels.tf (hunk 2 above)
Comment (tzarski0) [reply sdwan_profile_parcels.tf]: done in https://github.com/netascode/terraform-sdwan-nac-sdwan/pull/28/commits/a2066dc5b5b32f7dcade4fb6935a48139de78e4e

File: sdwan_profile_parcels.tf (hunk 1 above)
Comment (tzarski0) [reply sdwan_profile_parcels.tf]: done in https://github.com/netascode/terraform-sdwan-nac-sdwan/pull/28/commits/294cb9c0b01bb82cdd6359f0e163ae736a2a8abf

**Review by danischm — APPROVED** [review]

**Pull request merged**